MSAL/test/unit/MSALDeviceInfoProviderTests.m

```diff
@@ -278,6 +279,7 @@ - (void)testGetDeviceInfo_whenSSOExtensionPresent_andReturnedDeviceInfo_shouldRe
         XCTAssertNil(error);
         XCTAssertEqual(deviceInformation.deviceMode, MSALDeviceModeShared);
         XCTAssertEqualObjects(deviceInformation.extraDeviceInformation[@"isSSOExtensionInFullMode"], @"No");
+        XCTAssertEqualObjects(deviceInformation.extraDeviceInformation[@"mdm_id"], @"mdmId");
```

Comment:
nit: Use defined constant in test case as well 
XCTAssertEqualObjects(deviceInformation.extraDeviceInformation[MSID_BROKER_MDM_ID_KEY], @"mdmId");

Reply:
done